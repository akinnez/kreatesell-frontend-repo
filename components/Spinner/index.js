import {Spin} from 'antd';
import styles from './index.module.scss';

const Spinner = ({size = 'default'}) => (
	<div className={styles.spinnerWrapper}>
		<Spin size={size} />
	</div>
);

export default Spinner;
