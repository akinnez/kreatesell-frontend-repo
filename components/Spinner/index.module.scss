@use 'public/css/mixins';

.spinnerWrapper {
	height: 100%;
	@include mixins.flex(row, center, center);
}
