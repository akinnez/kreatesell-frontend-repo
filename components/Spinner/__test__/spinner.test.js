import renderer from "react-test-renderer";
import Spinner from "..";

it("renders a snapshot of <Spinner />", () => {
  const spinner = renderer.create(<Spinner />).toJSON();
  expect(spinner).toMatchSnapshot();
});
