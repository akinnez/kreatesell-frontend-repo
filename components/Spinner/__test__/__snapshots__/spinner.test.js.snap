// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <Spinner /> 1`] = `
<div
  className="spinnerWrapper"
>
  <div
    className="spinner"
  />
</div>
`;
